packages/loader/container-utils/src/error.ts

```diff
@@ -86,6 +86,18 @@ export class UsageError extends LoggingError implements IFluidErrorBase {
     }
 }
 
+/** Error indicating that a client's session has reached its time limit and is closed. */
+export class ClientSessionExpiredError extends LoggingError implements IFluidErrorBase {
+    readonly errorType = "clientSessionExpiredError";
```

Comment:
Why this is not ContainerErrorType.ContainerErrorType ?
I.e. why we have two places where same value is encoded?
@tyler-cai-microsoft  can you please follow up?

Reply:
Here's the issue for that follow up work: https://github.com/microsoft/FluidFramework/issues/8605,. We couldn't switch to the error right away as we needed the container-definitions change (the one where we add the error) to propagate down first.